### DISCUSSION TITLE
"Award" PMM Defaults

### DISCUSSION THREAD

**Discussion by benfishbus** [meisnate12/Plex-Meta-Manager#1192] [Ideas]
There are some things I perceive as gaps in the default collections for awards. I've developed my own collections to fill these gaps, but ultimately I think they are worth addressing directly in the PMM Defaults.

1. Some awards are only defined as year collections. Emmy and Independent Spirit awards, for example, don't have an overall "Winners" collection like Oscars does. This limits their usefulness, as year collections can take up a lot of space. (I turn off year collections where I can, such as Oscars.) All awards should include one or more all-years "Winners" collection.

2. Nominees are worth highlighting just as much as winners, in most cases. "Nominations" posters already exist in the images repository for many awards, and IMDb has search parameters for Oscar and Emmy noms. I'd like to see nominee collections for as many awards as possible. (I take the extra step of filtering out the winners, since there's already a collection highlighting them.)

**Comment by ckronengold** [comment]
Where did you post your collections? I'm all for enhancing the default. Agreed wholeheartedly that nominees should be available as an option, especially for categories like Best Picture. 

The other addition I'd like to see (or will try to mess with on my own) is the ability to choose specific award categories. I'd love to be able to select from Best Cinematography or Best Sound as well as Best Actor/Supporting Actor to name a few.
